packages/react/tsconfig.json

```diff
@@ -1,7 +1,7 @@
 {
   "extends": "../../tsconfig.json",
   "compilerOptions": {
-    "outDir": "lib"
+    "typeRoots": ["node_modules/@types"]
```

Comment:
Isn't this the default? Why do we need to specify this?

Reply:
Good call, looks like it's not needed and we can safely remove it.